pkg/jobcreator/controller.go

```diff
@@ -387,19 +387,22 @@ func (controller *JobCreatorController) downloadResult(dealContainer data.DealCo
 
 func (controller *JobCreatorController) acceptResult(deal data.DealContainer) error {
 	controller.log.Debug("Accepting results for job", deal.ID)
-	txHash, err := controller.web3SDK.AcceptResult(deal.ID)
-	if err != nil {
-		return fmt.Errorf("error calling accept result tx for deal: %s", err.Error())
-	}
-	controller.log.Debug("accept result tx", txHash)
+	// txHash, err := controller.web3SDK.AcceptResult(deal.ID)
+	// if err != nil {
+	// 	return fmt.Errorf("error calling accept result tx for deal: %s", err.Error())
+	// }
+	// controller.log.Debug("accept result tx", txHash)
```

Comment:
What would you think about removing the commented out code and instead leaving a comment stating what we did? Also could make it a TODO comment and suggest future work to fully remove this chain interaction.